decimal64.go

```diff
@@ -208,20 +199,19 @@ func roundStatus(significand uint64, exp int, targetExp int) discardedDigit {
 		return eq5
 	}
 	return gt5
-
 }
 
-// match scales matches the exponents of d and e and uses context to get the rounding status
-// contexts rounding method must be set or panic will occur
-func (context *roundContext) matchScales(d, e *decParts) {
+// match scales matches the exponents of d and e and returns the info about the discarded digit
+func matchScales(d, e *decParts)(discardedDigit) {
 	logicCheck(d.significand != 0, "d.significand (%d) != 0", d.significand)
 	logicCheck(e.significand != 0, "e.significand (%d) != 0", e.significand)
-
+	if d.exp == e.exp{
+	return eq0
```

Comment:
Bad indent.

Reply:
yep,for some reason linting stopped working

Reply:
VSCode? Mine stopped working recently too.